bin/cli.js

```diff
@@ -2,4 +2,14 @@
 require('source-map-support').install()
 
 const { MiraBootstrap } = require('../dist/src/cdk/bootstrap')
-new MiraBootstrap().initialize()
+
+async function run () {
+  try {
+    await new MiraBootstrap().initialize()
+  } catch (e) {
+    console.error(e)
```

Comment:
Hm, thinking that there should for sure be other try / catch blocks within `bootstrap.ts` an `app.ts`.  See `app.ts` around line # 164